_episodes/04-quality-control.md

```diff
@@ -108,15 +108,43 @@ will group at the ends of the sorted data. Sort your data by each field, one at
 attention to the top and the bottom of a column. 
 If your dataset is well-structured and does not contain formulas, sorting should never affect the integrity of your dataset.
 
-Let's try this with the *1980* tab in our messy spreadsheet. Go to that tab. Select Data then select Sort
-
-Sort by *wgt* in the order *Smallest to Largest*
+**Remember** to expand your sort in order to prevent data corruption. Expanding your sort ensures that the all the data in one row move together instead of only sorting a single column in isolation as shown in the screenshot.
 
 ![Figure of Sorting menu](../fig/sorting.png)
 
-- When you do this sort, do you notice anything strange?
+Let's try the following sorting exercise:
+
+> ## Exercise : 
+>
+> Download this data file based on the 2013 survey data to your computer: [survey_sorting_exercise](../data/survey_data_messy_quality_control.xlsx)
+>
```

Comment:
Thanks @zjsteyn for this great addition to the sorting lesson. If you could please update the link to the data file, I'll go ahead and merge this.